### ISSUE TITLE
Some character sheets do not open

### ISSUE THREAD

**Issue by p-falco** [moo-man/WFRP4e-FoundryVTT#432]
Hi - I noticed some character sheets do not open correctly in my setup. 
The one of Alexander Grun from Ubersreik adventures, for example, refuses to open, as do some others in "Bait and Witch". 
A PC's one also stopped opening - I am attaching two example json files. [Downloads.zip](https://github.com/moo-man/WFRP4e-FoundryVTT/files/6018810/Downloads.zip)

I am running 0.7.9 under docker, with some add-on modules, so if you need a list of the modules enabled I can provide it.

**Comment by moo-man** [comment]
What system version are you on?

**Comment by p-falco** [comment]
I updated all yesterday, so should be 3.4.1

(By the way, the job you're doing with WFRP is totally _GREAT_)

**Comment by moo-man** [comment]
Thanks! But please do verify your system version and ensure its 3.4.1

**Comment by p-falco** [comment]
I cannot even begin to say how sorry I am, but I always thought that updating all the modules also updated the game systems!!! So much that I had NEVER noticed the "Game Systems" tab!!! I was running a tremendously old version! I updated everything and now it works fine. 

I am horribly sorry for the waste of your time. Of course, the issue is totally invalid, and I closed it.  

Thanks for your patience, and again, truly superb work!!

![image](https://user-images.githubusercontent.com/47244113/108717328-73121b80-751d-11eb-8632-667887528828.png)
